llvm/lib/Transforms/Vectorize/LoopVectorize.cpp

```diff
@@ -9406,7 +9405,14 @@ static Instruction *lowerLoadUsingVectorIntrinsics(IRBuilderBase &Builder,
   }
   Call->addParamAttr(
       0, Attribute::getWithAlignment(Call->getContext(), Alignment));
-  return Call;
+  Instruction *Res = Call;
+  if (IsReverse) {
+    Value *BlockInMaskPart = Builder.getAllOnesMask(DataTy->getElementCount());
```

Comment:
Would be good to clarify the name here, the variable is not really for the block-in mask and independent of a part.